packages/taxdat/R/setup_helpers.R

```diff
@@ -602,6 +602,21 @@ check_do_parallel_prep <- function(do_parallel_prep,
   }
 }
 
+#' @include file_name_functions.R
+#' @title check_summary_admin_levels
+#' @description Checks whether the summary admin levels are valid
+#' @param summary_admin_levels the spatial level at which the statistics is summarized
+#' @return summary_admin_levels if valid
+#' @export
+check_summary_admin_levels <- function(summary_admin_levels) {
+  if (is.null(summary_admin_levels)) {
+    summary_admin_levels <- c(0,1,2)
+  } else {
+    summary_admin_levels <- try_conv_numeric(summary_admin_levels)
```

Comment:
Will this work on a vector? We may need to apply it to all vector elements in summary_admin_levels